openapi/openapi.yaml

```
@@ -508,6 +508,42 @@ components:
         - id
         - title
         - abbr
+    CapitalCommitment:
+      type: object
+      properties:
+        id:
+          type: string
+          description: A uuid used to refer to the capital commitment. 
+          format: uuid
+        type:
+          type: string
+          description: A four character string used to refer to the commitment type.
+          pattern: '^([A-z]{4})$'
+          example: DSGN
+        plannedDate:
+          type: date
```

Reading this back, I'm realizing I documented this incorrectly in the ticket. [Dates in openApi schema](https://swagger.io/docs/specification/data-models/data-types/) are type `string` with format `date`.

```suggestion
          type: string
          format: date
```